coalib/results/result_actions/PrintDebugMessageAction.py

```diff
@@ -15,7 +15,7 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
 
     def apply(self, result, original_file_dict, file_diff_dict):
         """
-        Print debug message
+        Print D(E)bug message
```

Comment:
i think its the 3rd time im asking and i cant see my comment anymore: why is this not "D"?